packages/tools/replay-tool/src/helpers.ts

```diff
@@ -0,0 +1,149 @@
+/*!
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { strict } from "assert";
+import fs from "fs";
+import { assert } from "@fluidframework/common-utils";
+import * as API from "@fluid-internal/client-api";
+import { IRequest } from "@fluidframework/core-interfaces";
+import { Container, Loader } from "@fluidframework/container-loader";
+import { TelemetryLogger } from "@fluidframework/telemetry-utils";
+import {
+    IDocumentServiceFactory,
+    IFluidResolvedUrl,
+    IResolvedUrl,
+    IUrlResolver,
+} from "@fluidframework/driver-definitions";
+import { IFileSnapshot } from "@fluidframework/replay-driver";
+import { getNormalizedSnapshot } from "@fluidframework/tool-utils";
+
+/**
+ * Helper function that normalizes the snapshot trees in the given file snapshot.
+ * @returns the normalized file snapshot.
+ */
+export function getNormalizedFileSnapshot(snapshot: IFileSnapshot): IFileSnapshot {
+    const normalizedSnapshot: IFileSnapshot = {
+        commits: {},
+        tree: getNormalizedSnapshot(snapshot.tree, { blobsToNormalize: [ "catchupOps" ] }),
```

Comment:
@anthony-murphy Note that I added "catchupOps" as blobs to normalize. The order of properties in catchupOps blob seems to have changed since 0.19.5 so the comparison was failing. It looked safe to normalize them since the contents was an object.

Reply:
Actually I am not 100% sure. The content is an array of messages. Its possible that the array order is important.
The reason why comparison is failing is because the order of properties in `ISequencedDocumentMessage` has changed.

Reply:
As per our discussion, I will remove this and update the order of these properties in the old snapshots.

Reply:
Updated the older snapshot for this - https://github.com/microsoft/FluidFrameworkTestData/pull/9